priv/eleveldb.schema

```diff
@@ -239,3 +239,31 @@
   {datatype, {enum, [true, false]}},
   hidden
 ]}.
+
+%% @doc Enable global expiry.  All leveldb databases / vnodes
+%% will use same expiry_minutes and whole_file_expiry settings.
+%% Same settings apply to all leveldb instances in multi_backend.
+{mapping, "leveldb.expiry_enabled", "eleveldb.expiry_enabled", [
+  {default, off},
+  {datatype, flag},
+  hidden
+]}.
+
+%% @doc Minutes until object expires.  Gives number of minutes
+%% a stored key/value will stay within the database before automatic
+%% deletion.  Zero disables expiry by age (minutes) feature
+{mapping, "leveldb.expiry_minutes", "eleveldb.expiry_minutes", [
+  {default, 0},
+  {datatype, integer},
```

Comment:
When you parse the expiry_minutes option in eleveldb.cc, you use the enif_get_ulong() function. Is that compatible with declaring it as an "integer" here? I'm not sure of the type system in this schema, which is why I'm asking (e.g., perhaps integers are always long and unsigned).